tests/scripts/all.sh

```diff
@@ -1843,6 +1844,8 @@ component_test_tls1_2_default_stream_cipher_only () {
     scripts/config.py unset MBEDTLS_GCM_C
     scripts/config.py unset MBEDTLS_CCM_C
     scripts/config.py unset MBEDTLS_CHACHAPOLY_C
+    #Disable TLS 1.3 (as no AEAD)
+    scripts/config.py unset MBEDTLS_SSL_PROTO_TLS1_3
```

Comment:
Note: this should have been caught by `check_config.h` rather than by runtime tests (I'm taking about the previous version of this PR where this component was building alright but had runtime failures). Can be done in a follow-up though.

Reply:
Yes I agree